### PR TITLE
Bump Version to 2.3.0

### PR DESCRIPTION
**Pull request by wbw20**
# Bump Version to 2.3.0

@zendesk/orchid 

Contains:
+ https://github.com/zendesk/ember-resource/pull/113

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-resource",
-  "version": "2.2.0",
+  "version": "2.1.1",
```
Comment (kencheeto) [inline bower.json]: ?

**Comment by roseleaf** [comment]
:+1:

File: bower.json (hunk 1 above)
Comment (wbw20) [inline bower.json]: Grrrr last person must have messed this up.  Bower is out of sync with package and changelog.

Comment (wbw20) [inline bower.json]: Im just going to go a big bump then, so that all package managers update to the new version properly.

Comment (wbw20) [inline bower.json]: this is why machines should do everything :computer:

**Comment by wbw20** [comment]
@kencheeto

**Comment by kencheeto** [comment]
:+1:

**Pull request merged**